docs/design/3_2_multirotor.tex

```diff
@@ -118,103 +115,54 @@ \subsubsection{Air Frame}\label{section:air-frame}
-most abundant size on the market for frames due to their versatility and are therefore less costly 
-and easy to repair/replace.
+Our ideal option is a X4 560 mm MMDS carbon fibre frame from JMT, as shown in Figure~\ref{fig:jmtx4}. The frame is carbon fiber to enhance structural rigidity while keeping the system extremely lightweight. While the frame also features foldable arms and landing gear for portability, we consider replacing the hinges with static parts to reduce dead weight in order to achieve a longer flight time, as part of our non-functional requirements (\textbf{NF.DR.7}). The X4 560 mm frame is also wide enough to support propollers up to 14 inches in diameter. See Section~\ref{section:propsys} for reasons why a larger propeller is generally better.
 
-Currently, the \textit{Upgrade F450} for CA\$30.00 and \textit{Diatone Q450} for CA\$20.00 from \textit{Banggood} are appealing options. They weigh between 300 g to 410 g, which are both acceptable.
+For our testing rig (which is also our back-up drone), we are using the \textit{Flamewheel F450} frame with 450 mm MMDS. This design has many clones and variants and thus extremely cheap to purchase and replace on the internet. The frame has a adequate balance between portability and capability. However, the 450 mm frames can only support at most 12 inch propellers.
 
-\subsubsection{Propulsion System}
+\subsubsection{Propulsion System}\label{section:propsys}
 
 The propulsion system covers the majority of the RPAS parts list and determines the flight performance of the RPAS. 
 
 \paragraph{Motor Type}
 
-The MRPAS is electrically powered with a battery (DC). Thus, we have two options for motors: DC brushed 
-motors and DC brushless motors.
-
-Brushed motors use mechanical brushes in contact with rotor’s commutator to switch polarity to sustain a 
-constant rotation, whereas the brushless motors use electronic controllers to switch the polarity in the 
-rotor electrically. 
+\underline{\textit{Brushed or brushless?}}
+The MRPAS is electrically powered with a battery (DC) and we have two options for motors: DC brushed  motors and DC brushless motors. Brushed motors use mechanical brushes in contact with rotor’s commutator to switch polarity to sustain a constant rotation, whereas the brushless motors use electronic controllers to switch the polarity in the rotor electrically. 
 
-For flight applications, we chose the brushless DC (BLDC) motors for their extended usable life time. 
-BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap 
-the polarity in the motor, which results in excess friction, heat, and potential sparking.
+For flight applications, we chose the brushless DC (BLDC) motors for their extended usable life time. BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap the polarity in the motor, which results in excess friction, heat, and potential sparking. 
```

Comment:
```suggestion
For flight applications, we chose the brushless DC (BLDC) motors for their extended usable lifetime. BLDC motors are also much more efficient than brushed motors because as they do not mechanically swap the polarity in the motor, which results in excess friction, heat, and potential sparking. 
```